tests/periph_gpio/Makefile

```diff
@@ -7,7 +7,19 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += benchmark
 
+BOARDS_BENCH_PORT_1 = \
+    slstk3402a \
+    #
+
+ifneq (,$(filter $(BOARDS_BENCH_PORT_1),$(BOARD)))
+  PORT_UNDER_TEST ?= 1
+else
+  PORT_UNDER_TEST ?= 0
+endif
+
 include $(RIOTBASE)/Makefile.include
 
+$(call target-export-variables,test,PORT_UNDER_TEST)
```

Comment:
Instead of this, another solution is to pass PORT_UNDER_TEST via CFLAGS and print the PORT under test from the application then use pexpect to read that value. I won't insist on that, just mentioning. Feel free to adapt, if you think it's better